kubetest/eks/eks.go

```diff
@@ -15,46 +15,48 @@ limitations under the License.
 */
 
 // Package eks implements 'kubetest' deployer interface.
-// It uses 'awstester' and 'kubectl' binaries, rather than importing
-// EKS packages directly. This is to eliminate the need of dependency
-// management, both in upstream and downstream.
+// It uses 'aws-k8s-tester' and 'kubectl' binaries, rather than importing internal packages.
```

Comment:
@BenTheElder Clarified that this package calls `aws-k8s-tester` binary as a subprocess rather than importing the underlying implementation (that requires AWS specific dependencies). In doing so, we can get away with dependency conflicts.

Reply:
Thanks! SGTM 👍